docs/getting-setup.md

```diff
@@ -0,0 +1,158 @@
+
+* Go [here](./debugging-the-debugger.md) if you want to start debugging the debugger!
+
+
+### Appendix
+
+#### Quick Setup
+
+This setup is for people on the DevTools team and DevTools wizards.
+
+```bash
+npm i -g yarn@0.16.1
+g clone git@github.com:devtools-html/debugger.html.git
+cd debugger.html
+yarn install
+
+# close firefox if it's already running
+/Applications/Firefox.app/Contents/MacOS/firefox-bin --start-debugger-server 6080 -P development
+
+# create a new terminal tab
+cd debugger.html
+yarn start
+```
+
+#### Starting Firefox
+
+If you're looking for an alternative to `yarn run firefox`, you have two options: cli, gcli.
+
+**Firefox CLI**
+
+1. run `firefox-bin` from the command line
+```bash
+/Applications/Firefox.app/Contents/MacOS/firefox-bin --start-debugger-server 6080 -P development
+```
+
+NOTE: You'll be shown a prompt to create a new "development" profile. The development profile is where your remote development user settings will be kept. *It's a good thing :)*
+
+2. Go to `about:config` and set these configs
+
+Navigate to `about:config` and accept any warning message. Then search for the following preferences and double click them to toggle their values to the following. [example](http://g.recordit.co/3VsHIooZ9q.gif)
+
+* `devtools.debugger.remote-enabled` to `true`
+* `devtools.chrome.enabled` to `true`
+* `devtools.debugger.prompt-connection` to `false`
+
+3. restart Firefox
+
+Close firefox and re-open it with the `firefox-bin` command.
+
+
+#### Starting Chrome
+
+There are two ways to run chrome. Here's the easy way to run chrome
+
+```bash
+yarn run chrome
+```
+
+Here's the slightly harder way.
+
+```bash
+ /Applications/Google\ Chrome.app/Contents/MacOS/Google\ Chrome --remote-debugging-port=9222 --no-first-run --user-data-dir=/tmp/chrome-dev-profile
+```
+
+NOTE: that the [script](../bin/chrome-driver) just automates the command :)
+
+#### Starting Node
+
+It's easy to start Node in a mode where DevTools will find it:
+
+* *--inspect* - tells node to open a debugger server
+* *--inspect=9223* - tells node to open a debugger server on 9223 instead of 9229.
+* *--debug-brk* - tells node to pause on the first statement
+
+```bash
+node --inspect --debug-brk ./node_modules/.bin/webpack
+```
+
+NOTE: *./node_modules/.bin/webpack* could be anything. We're often debugging webpack these days so it's often appropriate :/
+
+**Note:** Currently Node.js debugging is limited in some ways, there isn't support for seeing variables or the console, but you can manage breakpoints and navigate code execution (pause, step-in, step-over, etc.) in the debugger across various sources.
+
+#### Windows + Linux setup
+
+Windows and Linux should *just work*, but unfortunately there are several edge cases.
+
+If you find any issues on these two platforms comment on these issues:
+* [windows](https://github.com/devtools-html/debugger.html/issues/1248)
+* [linux](https://github.com/devtools-html/debugger.html/issues/1082)
+
+
+```
```

Comment:
ughh, meant to have some context here. not sure if we need the same linux details